### PR TITLE
Add hoverxref extension

### PR DESCRIPTION
**Pull request by Daltz333**
Very useful for state-space glossary terms.

### CODE REVIEW

File: source/requirements.txt [hunk 1]
```diff
@@ -9,3 +9,4 @@ sphinx-notfound-page==0.4
 transifex-client==0.13.9
 sphinx-intl==2.0.1
 sphinxcontrib-svg2pdfconverter==1.0.1
+-e git+https://github.com/readthedocs/sphinx-hoverxref.git@3558a67e56bf27f08d52425bbcd1d63989c6224e#egg=hoverxref.extension
```
Comment (AustinShalit) [inline source/requirements.txt:12]: Do they not work with the latest version?

Comment (Daltz333) [reply source/requirements.txt:12]: It's not on pypi yet.

Comment (AustinShalit) [reply source/requirements.txt:12]: https://pypi.org/project/sphinx-hoverxref/#files

Comment (Daltz333) [reply source/requirements.txt:12]: They haven't published a release with term support yet.

**Pull request merged**